store/cachemulti/store.go

```diff
@@ -122,8 +123,21 @@ func (cms Store) GetStoreType() types.StoreType {
 // Write calls Write on each underlying store.
 func (cms Store) Write() {
 	cms.db.Write()
+	eg := new(errgroup.Group)
 	for _, store := range cms.stores {
```

Comment:
should there be a concurrency limit? or is it good enough to delegate this work completely to the go scheduler?

Reply:
we should do some benchmarks with databases and come up with a optimal number of runners